src/Classes/ServiceAPI/MyRadio_BannerCampaign.php

```diff
@@ -494,44 +239,26 @@ public static function getCampaignLocations()
     /**
      * Returns the form needed to create or edit Banner Campaigns.
      *
-     * @param int $bannerid The ID of the Banner that this Campaign will be/is linked to
+     * @param int $newBannerID The ID of the Banner that this Campaign will be created for
+     * @param int $availabilityid The ID of the campaign that will be edited
      *
      * @return MyRadioForm
      */
-    public static function getForm($bannerid = null)
+    public static function getForm($availabilityid = null, $newBannerID = null)
```

Comment:
I think use of camelCase for variable names is inconsistent with everything else (although tbf everything else is a mixture of _ and no _)